src/main/resources/org/jenkinsci/plugins/registry/notification/opt/impl/TriggerOnSpecifiedImageNames/config.groovy

```diff
@@ -25,7 +25,14 @@ package org.jenkinsci.plugins.registry.notification.opt.impl.TriggerOnSpecifiedI
 
 def st = namespace("jelly:stapler")
 def f = namespace(lib.FormTagLib)
+def l = namespace("/lib/local")
 
-f.entry(title:_("Repositories"), field: "repoNames") {
-    f.expandableTextbox(value: instance?.repoNames?.join("\n"))
+if (context.getVariableWithDefaultValue("divBasedFormLayout", "false") == "true") {
```

Comment:
I do not understand this.

if `divBasedFormLayout` is true, then we use the form taglib from Jenkins (this is a newer version of Jenkins that has switched).
Otherwise - we use our own which is a fork of the older jenkins taglib?   but if its not enabled - then what we have copy and pasted (by the looks of it) would be exactly what Jenkins has provided us with?

that is on cores before the div switch,   `l.entry` is the same as `f.entry`?

Reply:
it's not the same, the differences are listed in the PR description,

there's custom css applied and they call forked tags in some cases